apps/client/src/features/rundown/Rundown.tsx

```diff
@@ -29,6 +28,44 @@ import style from './Rundown.module.scss';
 
 const RundownEntry = lazy(() => import('./RundownEntry'));
 
+type KeyCombo = { altKey: boolean; ctrlKey: boolean; shiftKey: boolean; code: string };
+const checkKeys = (event: KeyboardEvent, keyCombo: KeyCombo) => {
+  return (
+    keyCombo.altKey === event.altKey &&
+    keyCombo.ctrlKey === event.ctrlKey &&
+    keyCombo.shiftKey === event.shiftKey &&
+    keyCombo.code === event.code
+  );
+};
+
+const keyCombos = {
+  selectUp: (event: KeyboardEvent) =>
+    checkKeys(event, { altKey: true, ctrlKey: false, shiftKey: false, code: 'ArrowUp' }),
+  selectDown: (event: KeyboardEvent) =>
+    checkKeys(event, { altKey: true, ctrlKey: false, shiftKey: false, code: 'ArrowDown' }),
+  reorderUp: (event: KeyboardEvent) =>
+    checkKeys(event, { altKey: true, ctrlKey: true, shiftKey: false, code: 'ArrowUp' }),
+  reorderDown: (event: KeyboardEvent) =>
+    checkKeys(event, { altKey: true, ctrlKey: true, shiftKey: false, code: 'ArrowDown' }),
+  addEventUp: (event: KeyboardEvent) =>
+    checkKeys(event, { altKey: true, ctrlKey: false, shiftKey: true, code: 'KeyE' }),
+  addEventDown: (event: KeyboardEvent) =>
+    checkKeys(event, { altKey: true, ctrlKey: false, shiftKey: false, code: 'KeyE' }),
+  addDelayUp: (event: KeyboardEvent) =>
+    checkKeys(event, { altKey: true, ctrlKey: false, shiftKey: true, code: 'KeyD' }),
+  addDelayDown: (event: KeyboardEvent) =>
+    checkKeys(event, { altKey: true, ctrlKey: false, shiftKey: false, code: 'KeyD' }),
+  addBlockUp: (event: KeyboardEvent) =>
+    checkKeys(event, { altKey: true, ctrlKey: false, shiftKey: true, code: 'KeyB' }),
+  addBlockDown: (event: KeyboardEvent) =>
+    checkKeys(event, { altKey: true, ctrlKey: false, shiftKey: false, code: 'KeyB' }),
+  copy: (event: KeyboardEvent) => checkKeys(event, { altKey: false, ctrlKey: true, shiftKey: false, code: 'KeyC' }),
+  pastUp: (event: KeyboardEvent) => checkKeys(event, { altKey: false, ctrlKey: true, shiftKey: true, code: 'KeyV' }),
+  pastDown: (event: KeyboardEvent) => checkKeys(event, { altKey: false, ctrlKey: true, shiftKey: false, code: 'KeyV' }),
+  deleteEntry: (event: KeyboardEvent) =>
+    checkKeys(event, { altKey: true, ctrlKey: false, shiftKey: false, code: 'Backspace' }),
+} as const;
+
```

Comment:
@cpvalente I tried to make the shortcut definitions more manageable
 do you think this is OK?